src/main/generated/org/sopt/domain/QMember.java

```diff
@@ -0,0 +1,48 @@
+package org.sopt.domain;
+
+import static com.querydsl.core.types.PathMetadataFactory.*;
+
+import com.querydsl.core.types.dsl.*;
+
+import com.querydsl.core.types.PathMetadata;
+import javax.annotation.processing.Generated;
+import com.querydsl.core.types.Path;
+import com.querydsl.core.types.dsl.PathInits;
+
+
+/**
+ * QMember is a Querydsl query type for Member
+ */
+@Generated("com.querydsl.codegen.DefaultEntitySerializer")
+public class QMember extends EntityPathBase<Member> {
+
+    private static final long serialVersionUID = 1323137632L;
+
+    public static final QMember member = new QMember("member1");
+
+    public final ListPath<Article, QArticle> articles = this.<Article, QArticle>createList("articles", Article.class, QArticle.class, PathInits.DIRECT2);
+
+    public final DatePath<java.time.LocalDate> birthDate = createDate("birthDate", java.time.LocalDate.class);
+
+    public final StringPath email = createString("email");
+
+    public final EnumPath<org.sopt.domain.enums.Gender> gender = createEnum("gender", org.sopt.domain.enums.Gender.class);
```

Comment:
```suggestion
    public final EnumPath<Gender> gender = createEnum("gender", Gender.class);
```
`import org.sopt.domain.enums.Gender` 사용하면 더 깔끔하게 나타낼 수 있을 것 같아용